### DISCUSSION TITLE
Project added but not visible

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [alefragnani/vscode-project-manager#602] [Q&A] [answered]
Hello,
I add a new project by "Save Project", the project seems to be saved (from the tooltip message) but I can't see it on the sidebar.
In fact the sidebar seems to be filtered
![image](https://user-images.githubusercontent.com/412061/167677513-1c43f2be-16a7-4bf6-93e2-06b403d8c256.png)

I click on any of those buttons to enable/disable the filters but nothing is visible. 
If I click on tags it tells me: "No tags Available/ Defined" but I have this on my settings.json
```
"projectManager.tags": [
    "Personal",
    "Work",
  ],
```
If I try to save the current project as suggested it telles me that the current projet is already saved.
What am I doing wrong?

**Comment by alefragnani** [comment, marked as answer]
Hi @LeonardoGentile ,

Maybe you are facing #557 or #594, and if you see the `filtered by tags`, the extension has some filter applied.

Just to be aware, the tags you define in `projectManager.tags` settings are just _suggestions_, and will be available to you when you try to apply a tag to some project. But, you can define any tag, for any project, if you edit the projects manually (using the `Project Manager: Edit Projects` command, which will open the `projects.json` file. 

So, I suggest you to:

* Edit the `projects.json` file
* Remove the tags, manually. 
* Save the `projects.json` file
* Click in the `Edit tags` button and _select all tags_ (which should display only the ones you previously configured via `projectManager.tags` setting.

This should display the projects you have saved

Hope this helps

**Reply by LeonardoGentile** [reply]
Thanks 👍
Unfortunately nothing you suggested fixed it.

I've completely remove the extensions and all settings from my settings.json.
I've remove the projects.json as well.

I've re-installed the extension, supposed to be clean but the same behavior persist.

When I save a project it tell me the projects are filtered by tags and I can't see it on the sidebar.
If I inspect the projects.json the `tags` field is an empty array 🤷‍♂️

**Reply by alefragnani** [reply]
What happens when you hit the `Filter by Tags` button? If it does not work, and display a message about `No tags defined/available`, add some tag to the `Tags` entry in `projects.json`. There is an issue about that filtering.

Also, instead of _select all tags_ when using the `Filter by Tags` button, **unselect all** (clicking the checkbox at left of the input box. Just noticed I posted the wrong instructions.

Unfortunately, the filtering state is stored in VS Code internal storage, which is not available to fix easily anymore. It will depend on extension's command.

**Reply by LeonardoGentile** [reply]
That fixed it, indeed!
Thanks a lot 👍

**Reply by alefragnani** [reply]
That's great to hear!
